lib/xboxkrnl/xboxkrnl.h

```diff
@@ -9,6 +9,7 @@
 #pragma once
 
 #include <xboxkrnl/xboxdef.h>
+#include <xboxkrnl/xboxnt.h>
```

Comment:
These headers need the same setup that's being done here, to get proper struct padding etc.?

Reply:
Upstreamed